scripts/release-notes.py

```diff
@@ -16,6 +16,7 @@
 import urllib.parse
 from os.path import expanduser
 import sys
+from github import Github
```

Comment:
I think the nice feature of the existing script is that it runs without extra dependencies that need to be pip-installed